include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -926,24 +926,24 @@ struct get_turn_info_type<Geometry1, Geometry2, AssignPolicy, Tag1, Tag2, linear
     : overlay::get_turn_info_linear_areal<AssignPolicy>
 {};
 
-template <typename Geometry1, typename Geometry2, typename SegmentRatio,
+template <typename Geometry1, typename Geometry2, typename Point, typename SegmentRatio,
           typename Tag1 = typename tag<Geometry1>::type, typename Tag2 = typename tag<Geometry2>::type,
           typename TagBase1 = typename topological_tag_base<Geometry1>::type, typename TagBase2 = typename topological_tag_base<Geometry2>::type>
-struct turn_operation_type
+struct define_turn_operation_type
```

Comment:
What is the reasoning behind this change? Verbs are typically used for functions or metafunctions semantically modifying a type. Consider that `point_type` is not named `define_point_type`.

Reply:
What different `turn_operation_type` are you talking about? I tried to find it but I only see typedefs. AFAIU this is the only `struct` of this name. So if my understanding is correct this is the same case as it is with `geometry::point_type<>::type` and `typedef ... point_type`. Or am I missing something?

Reply:
Yes, it’s probably the only struct indeed. All right, I will rename it back, it’s easier than it was to make it unique.